flutter/lib/src/widgets_binding_observer.dart

```diff
@@ -66,7 +66,7 @@ class SentryWidgetsBindingObserver with WidgetsBindingObserver {
     if (!_options.enableWindowMetricBreadcrumbs) {
       return;
     }
-    final window = WidgetsBinding.instance.window;
+    final window = WidgetsBinding.instance!.window;
```

Comment:
This breaks if `WidgetsBinding.instance == null`. There are couple more cases which are similar.